.github/workflows/functional.yml

```diff
@@ -115,10 +123,23 @@ jobs:
            path: rstuf-umbrella
            ref: main
 
+      - name: Deploy online.key to Docker Volume
+        # This is an workaround for the issue https://github.com/vmware/repository-service-tuf-worker/issues/241
+        # Once solving it:
+        # 1. Change workers to use secrets with the online key
+        # 2. Remove this step
+        run: docker cp tests/files/key_storage/online.key "${{ job.services.rstuf-worker-1.id }}:/var/opt/repository-service-tuf/keystorage/"
```

Comment:
Wouldn't this copy the only key only to the first RSTUF worker instance? 
Don't we want to copy it to the second as well?

Reply:
Yes, you are right.
I just didn't know containers could share a volume.